### ISSUE TITLE
extra parameter in shader models/mapobjects/monitor/display_roq_03

### ISSUE THREAD

**Issue by ghost** [Masmblr/map-Nova_src#2]
I think this entry have an extra parameter. It looks like ioquake3 does not notices it, but does not reads it neither (I have noticed the same thing in tremor-2016, so opening issue as I now wonder if this is a typo or if something is expected to happen).

https://github.com/Masmblr/map-Nova_src/blob/8f831b470b6193f286a92cfdc4b48f6873ce8516/1.1/scripts/nova.shader#L1218

**Comment by Masmblr** [comment]
I've reviewed the line and tested the current release, and it seems like everything is working correctly. The line you mentioned is used to make the loading arrows rotate. They rotate in-game during the briefing. I've added a screenshot. The graphic file 'load.jpg' is the rotating arrow image. Did you use a vanilla Tremulous installation? Some problems are caused by too many conflicting pk3 -files.
![shot0000](https://github.com/Masmblr/map-Nova_src/assets/93756033/0440e6d3-808d-49d1-81ae-90150d45df0d)
